### ISSUE TITLE
pbench-iozone needs -+m and -+h options for cluster processing

### ISSUE THREAD

**Issue by wilkins11** [distributed-system-analysis/pbench#352]
iozone has a cluster mode that allows the user to specify a master host (-+h) as well as a config file (-+m). The config file contains an entry for each target client that includes the client name, the target directory (the -f option), and the path to the iozone executable.

   vm1 /mnt/ceph/ioz /usr/local/bin/iozone
   vm2 /mnt/ceph/ioz /usr/local/bin/iozone
   vm3 /mnt/ceph/ioz /usr/local/bin/iozone
      ...

This cluster option automatically controls the sync'd start/stop of all clients and aggregates the results very conveniently.

**Comment by wilkins11** [comment]
Plz back-burner this request while we see if pbench-fio will suit for sequential testing.

**Comment by bengland2** [comment]
@wilkins11  , fio --client will do the equivalent of iozone -+m with a couple of minor exceptions described in issue #353 .  BTW, iozone is difficult to change, fio is much better maintained and has a more active community so I think we are better off in the long run transitioning to fio.

**Comment by bengland2** [comment]
I agree, can we drop pbench-iozone?  iozone is an unmaintainable mess because of legacy code (had to support all UNIX variants back in the 90s), fio has a real community and will let us implement changes to it if needed.
